src/main/routes.ts

```diff
@@ -292,6 +294,10 @@ export class Routes {
           errorHandler(new SafetyConcernsPostController(step.form.fields).post)
         );
         app.post(RESPONDENT_CHECK_ANSWERS_NO, errorHandler(new SafetyConcernsPostController(step.form.fields).post));
+        app.post(
```

Comment:
@giorgi-hmcts Why this route is require ?
We can handle navigation as part of citizenSequence.ts right ?